### PR TITLE
Removes 'none' classes after selection change

### LINKED ISSUE #658: "none" classes not removed when selection="after" is applied to existing slider
`.slider-track-low`, `.slider-selection`, and `.slider-track-high` elements maintain their `hide` class after a change to the slider's selection attribute. I will post a PR fix and tests shortly.

The first slider below displays correctly (select="after" defined at init). The second slider had its selection attribute changed, but does not update its appearance.
image

https://fiddle.jshell.net/jvx1fgof/2/

### PR DESCRIPTION
**Pull request by john-clarke**
Fixes #658

### CODE REVIEW

File: src/js/bootstrap-slider.js [hunk 1]
```diff
@@ -689,6 +689,12 @@ const windowIsDefined = (typeof window === "object");
 				this._addClass(this.trackHigh, 'hide');
 			}
 
+      else if (this.options.selection === 'after' || this.options.selection === 'before') {
```
Comment (seiyria) [inline src/js/bootstrap-slider.js]: please fix the indentation here

**Comment by seiyria** [comment]
looks good to me. @rovolution ?

**Comment by rovolution** [comment]
this is a great PR! Thank again for making this very high-quality PR @john-clarke!

**Comment by john-clarke** [comment]
@rovolution Glad it worked out. bootstrap-slider has been a very useful package to us this year. This PR was just a small contribution to your work. Thank you for keeping it going!

**Comment by rovolution** [comment]
merged and published to 9.5.1. Thanks again!

**Pull request merged**